### ISSUE TITLE
Slow hierarchy visualization

### ISSUE THREAD

**Issue by jirihelmich** [linkedpipes/visualizations#50]
[atc.zip](https://github.com/linkedpipes/visualizations/files/203878/atc.zip)

**Comment by jirihelmich** [comment]
```
! @6pjhe6c5m - Internal server error, for (GET) [/api/v1/skos/schemes/156?tolerant=false] ->

play.api.Application$$anon$1: Execution exception[[MatchError: model.rdf.LocalizedValue@47b037b1 (of class model.rdf.LocalizedValue)]]
    at play.api.Application$class.handleError(Application.scala:296) ~[com.typesafe.play.play_2.11-2.3.7.jar:2.3.7]
    at play.api.DefaultApplication.handleError(Application.scala:402) [com.typesafe.play.play_2.11-2.3.7.jar:2.3.7]
    at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$14$$anonfun$apply$1.applyOrElse(PlayDefaultUpstreamHandler.scala:205) [com.typesafe.play.play_2.11-2.3.7.jar:2.3.7]
    at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$14$$anonfun$apply$1.applyOrElse(PlayDefaultUpstreamHandler.scala:202) [com.typesafe.play.play_2.11-2.3.7.jar:2.3.7]
    at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36) [org.scala-lang.scala-library-2.11.7.jar:na]
Caused by: scala.MatchError: model.rdf.LocalizedValue@47b037b1 (of class model.rdf.LocalizedValue)
    at scala.PartialFunction$$anon$1.apply(PartialFunction.scala:253) ~[org.scala-lang.scala-library-2.11.7.jar:na]
    at scala.PartialFunction$$anon$1.apply(PartialFunction.scala:251) ~[org.scala-lang.scala-library-2.11.7.jar:na]
    at scala.PartialFunction$Unlifted.applyOrElse(PartialFunction.scala:233) ~[org.scala-lang.scala-library-2.11.7.jar:na]
    at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:36) [org.scala-lang.scala-library-2.11.7.jar:na]
    at play.api.libs.json.OWrites$$anon$1$$anonfun$contramap$1.apply(Writes.scala:56) ~[com.typesafe.play.play-json_2.11-2.3.7.jar:2.3.7]
```

**Comment by jirihelmich** [comment]
Fixed in e25750e.
